Pipfile

```diff
@@ -7,7 +7,7 @@ name = "pypi"
 
 [packages]
 
-flask = "==0.12.2"
+flask = "*"
```

Comment:
I'm not sure that always using the latest version of Flask is the best approach. Pinned versions are reliable.

Reply:
Will do.
When you do `pipenv install "package-name"` it always adds the package to the pipfile like this by default so that's where this has come from